### ISSUE TITLE
Failure configuring with internal libabt

### ISSUE THREAD

**Issue by devreal** [pmodels/bolt#59]
Trying to build BOLT with internal Argobots on current `master` results in a CMake error:

```
$ cmake ../ -DCMAKE_INSTALL_PREFIX=$HOME/opt/bolt-git -DLIBOMP_USE_ARGOBOTS=on
...
CMake Error at /usr/share/cmake-3.10/Modules/ExternalProject.cmake:2474 (message):
  No download info given for 'libabt' and its source directory:

   ~/src/bolt/git/bolt/external/argobots

  is not an existing non-empty directory.  Please specify one of:

   * SOURCE_DIR with an existing non-empty directory
   * DOWNLOAD_COMMAND
   * URL
   * GIT_REPOSITORY
   * SVN_REPOSITORY
   * HG_REPOSITORY
   * CVS_REPOSITORY and CVS_MODULE
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/ExternalProject.cmake:3029 (_ep_add_download_command)
  external/CMakeLists.txt:32 (ExternalProject_Add)
```

**Comment by shintaro-iwasaki** [comment]
If you download BOLT via GitHub, you need to download the internal Argobots by using `git submodule`.
```
git submodule init
git submodule update
```
I believe it should solve the issue.

**Comment by devreal** [comment]
Ahh yes, that did the trick. That is something CMake can take care of though (or at least give a hint on what to do)

**Comment by shintaro-iwasaki** [comment]
Thank you for the suggestion! It has been fixed in #60.

I will close this issue.
